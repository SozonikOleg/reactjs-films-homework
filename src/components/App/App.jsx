/* eslint-disable no-undef */
import React from 'react';
import { render } from 'react-dom';
import { Provider } from 'react-redux';
import { hot } from 'react-hot-loader/root';
import { BrowserRouter as Router } from 'react-router-dom';
import styles from './App.module.scss';
import Header from '../Header';
import Main from '../Main';
import store from '../../modules/store';

function App() {
  return (
    <Router>
      <div className={styles.three}>
        <Header />
        <Main />
      </div>
    </Router>
  );
}

render(
  <Provider store={store}>
    <App />
  </Provider>,
  document.getElementById('root'),
);

export default hot(App);
