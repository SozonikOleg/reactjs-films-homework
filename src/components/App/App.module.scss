.three {
    @import url('https://fonts.googleapis.com/css?family=Josefin+Sans');
    font-family: 'Bungee Inline', cursive;
    color: green;
}