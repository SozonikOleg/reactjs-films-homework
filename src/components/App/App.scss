*{
    margin: 0px;
    padding: 0px;
}

.three {
    @import url('https://fonts.googleapis.com/css?family=Josefin+Sans');
    font-family: 'Bungee Inline', cursive;
    color: green;
}