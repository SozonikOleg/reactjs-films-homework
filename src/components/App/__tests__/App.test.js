import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import App from '../App';

it('ErrorBoundary renders', () => {
  const props = {
    location: { search: '' },
  };
  const renderer = new ShallowRenderer();
  const result = renderer.render(
    <App {...props} />,
  );
  expect(result).toMatchSnapshot();
});
