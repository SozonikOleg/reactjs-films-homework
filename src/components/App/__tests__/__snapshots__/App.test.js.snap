// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorBoundary renders 1`] = `
<ErrorBoundary>
  <BrowserRouter>
    <div
      className="three"
    >
      <Memo(ConnectFunction) />
      <Main />
      <Footer />
    </div>
  </BrowserRouter>
</ErrorBoundary>
`;
